### PR TITLE
Styling and routing

### PR DESCRIPTION
**Pull request by jennydely**
Added some styling
integrate the formulars into routes

### CODE REVIEW

File: src/GlobalStyles.js [hunk 1]
```diff
@@ -51,37 +51,39 @@ export default createGlobalStyle`
...
+    padding: 11px;
+ }
 
-select {
+  select {
   min-width:fit-content;
-  min-height:fit-content;
   font-size: 140%;  
-  padding-top: 4px;
+  padding: 4px 25px 0px 4px;
```
Comment (soerenstein) [inline src/GlobalStyles.js:85]:
```suggestion
  padding: 4px 25px 0 4px;
```

File: src/Pages/Packlist/usePacklists.js [hunk 2]
```diff
@@ -21,8 +21,9 @@ export default function usePacklists() {
     putPacklist(packlist)
       .then((packlistUpdate) => {
         const index = packlists.findIndex(
-          (packlist) => packlist.id === packlistUpdate.id
+          (packlist) => '' + packlist.id === '' + packlistUpdate.id
         )
+        console.log('haha', index, packlistUpdate, packlists)
```
Comment (soerenstein) [inline src/Pages/Packlist/usePacklists.js:26]:
```suggestion

```

**Review by soerenstein — COMMENTED** [review]
Good job. 👏✅  Just two small suggestions.

File: src/GlobalStyles.js (hunk 1 above)
Comment (jennydely) [reply src/GlobalStyles.js:85]: oh yes ;D forgot to delete the px...

File: src/Pages/Packlist/usePacklists.js (hunk 2 above)
Comment (jennydely) [reply src/Pages/Packlist/usePacklists.js:26]: :D still forget console.logs . thanks

**Pull request merged**